lib/aws-s3.js

```diff
@@ -369,10 +294,13 @@ module.exports = class AWSAttachmentsService extends require("./object-store.js"
     if (req?.data?.content) {
       const response = await SELECT.from(req.target, { ID: targetID }).columns("url")
       if (response?.url) {
+        const bucket = await this.getBucket();
+        const client = await this.getClient();
```

Comment:
how about this: (Also semicolons can be safely removed)

```suggestion
        const { bucket, client } = await this.createClient()
```